.github/actions/install-debian-packages/action.yml

```diff
@@ -16,18 +12,25 @@ inputs:
 runs:
   using: "composite"
   steps:
+    - name: Cache debian packages
+      uses: actions/cache@v2
+      id: cache-debs
+      with:
+        path: "~/cache-debs"
+        # Update cache key if you add or update a package.
+        key: v3
```

Comment:
One additional question here. What is the procedure to update a dependency? Or better, how avoid this logic to always use the check instead of trying to install new versions of the libraries?

Reply:
As long as we do not change the cache key, we will be stuck with the version of the software which were installed for this given cache key.
This should not be a problem for `libbpf` and `libbseccomp` as we install specific version of this libraries, but for `llvm` it can be more troublesome...
We will need to think to a better way to handle this case while not losing the speed gain of the cache.

Reply:
You're right. Let's think about a better way to handle it in another PR.